I/ipx/build_tarballs.jl

```diff
@@ -0,0 +1,57 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ipx"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ERGO-Code/ipx.git", "234d484ed1813ce1d95e49c2d86da79d2df18f95"),
+    GitSource("https://github.com/ERGO-Code/basiclu.git", "fe63cd34ab0259329e5a979df7f19ff0de59cfae"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cp $WORKSPACE/srcdir/basiclu/include/* "${includedir}/."
+cd $WORKSPACE/srcdir/ipx/
+atomic_patch -p1 "${WORKSPACE}/srcdir/patches/example.patch"
+ARGS=(CXX="c++ -std=c++11" BASICLUROOT="$prefix")
+if [[ ${target} == *mingw32* ]]; then
+    ARGS+=(UNAME="Windows" SO_OPTS="-shared")
+else
+    ARGS+=(UNAME="$(uname)")
+fi
+make "${ARGS[@]}"
+cp lib/* "${libdir}/."
+cp include/* "${includedir}/."
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux", libc = "glibc"),
+    Platform("aarch64", "linux", libc = "musl"),
+    Platform("powerpc64le", "linux"),
+    Platform("x86_64", "freebsd")
+]
```

Comment:
This should be the same as
```suggestion
platforms = filter!(p -> (Sys.islinux(p) || Sys.isfreebsd(p)) && nbits(p) == 64, supported_platforms())
```
?